src/Components/Dashboard/Feeds.js

```diff
@@ -1,110 +1,100 @@
-import React from 'react';
+import React, { useState } from 'react';
 import Like from '../Assets/like.png';
 import isLike from '../Assets/liked.png';
 import moment from 'moment';
 import commentIcon from '../Assets/comment.png';
-import LoadMore from './LoadMore';
+import { EmojiEmotionsOutlined } from '@material-ui/icons';
 import { getBackgroundColor } from '../../Utils/colors';
-import { ColorRing } from 'react-loader-spinner';
+import { ColorRing, Oval } from 'react-loader-spinner';
 import Unavailiabe from '../../Utils/unavailiable1.png';
+import InfiniteScroll from 'react-infinite-scroll-component';
+import { getFeedDetails } from '../../Auth/Actions/feedActions';
+import { useDispatch } from 'react-redux';
+
+const Feeds = ({
+    feeds,
+    setData,
+    setPostArticle,
+    setPostArticleModal,
+    offSet,
+    setOffSet,
+    feedsLength,
+    requesting,
+}) => {
+    let pageYOffset = window.pageYOffset;
+    // console.log(feeds);
+    const dispatch = useDispatch();
+    const [hasMore, setHasMore] = useState(true);
+    const fetchMoreData = () => {
+        if (feedsLength >= 10) {
```

Comment:
this check should be against the total number of items in the db and the total number of items you are rendering